### DISCUSSION TITLE
PRTouch_v2 Configuration Documentation?

### DISCUSSION THREAD

**Discussion by sleewok** [CrealityOfficial/Creality_Sonic_Pad#104] [Q&A]
Is there any documentation available for the Klipper config parameters of the PRTouch_v2? I would like to make some tweaks but I am unable to find any information on what the various parameters actually do.

**Comment by lancer234** [comment]
> Is there any documentation available for the Klipper config parameters of the PRTouch_v2? I would like to make some tweaks but I am unable to find any information on what the various parameters actually do.

Did you have any luck with this?

**Comment by sleewok** [comment]
Nope. I ended up disabling the prtouch and installed my old CR-Touch on my
K1C. It is faster and more accurate by far. Shaves over 5 minutes off the
print start too.

**Reply by FxWoody** [reply]
i have a KE and need to disable the PrTouch and Bltouch too ... what did you do exactly ??? Just # every lines in printer.cfg ??? Because i still end up with errors in Mainsail :( Want to install Beacon instead ;)

**Comment by sleewok** [comment]
I think there are some related python files that need modified or removed.
I don't recall which ones.

**Comment by Chris77123** [comment]
> Is there any documentation available for the Klipper config parameters of the PRTouch_v2? I would like to make some tweaks but I am unable to find any information on what the various parameters actually do.

?

**Reply by sleewok** [reply]
I still haven't found anything. It's a shame that Creality has not released the source code for it (AFAIK). There's a way to wire the load cells in parallel (so that the controller sees them as one loadcell) and have klipper handle the loadcell.  At least for the K1/K1C (https://github.com/cryoz/K1_tenso_manual/blob/main/README_ENG.md). 

I haven't tried it yet. The required klipper branch is a bit behind mainline and I need the version that supports Orca Slicer's Adaptive Pressure Advance.
